pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -2265,7 +2265,7 @@ public void testFailedUpdatePartitionedTopic() throws Exception {
 
         try {
             admin.topics().updatePartitionedTopic(partitionedTopicName, newPartitions, false, false);
-        } catch (PulsarAdminException.PreconditionFailedException e) {
+        } catch (Exception e) {
```

Comment:
Does the exception has changed?

Reply:
Yes, as we change `updatePartitionedTopic` from sync to async, the exception is wrapped by CompletionException

Reply:
I didn't call e.getCause() in `resumeAsyncResponseExceptionally(asyncResponse, ex);`
Thanks, fixed.